src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -739,6 +739,35 @@
 			"# For example, to give a `visemayor` preference over an `assistant`, change this parameter to `visemayor,assistant`."
 	),
 	
+	GTOWN_SETTINGS_COMMAND_BLACKLISTING(
+			"global_town_settings.town_command_blacklisting",
+			"",
+			"",
+			"# Allows blocking commands inside towns and limiting them to plots owned by the players only.",
+			"# Useful for limiting sethome/home commands to plots owned by the players themselves and not someone else."
+	),
+	GTOWN_SETTINGS_ENABLE_COMMAND_BLACKLISTING(
+			"global_town_settings.town_command_blacklisting.enabled",
+			"true",
+			"",

```

Comment:
The default should probably be set to false, some servers might still want players to be able to use sethome and home by default. (line 764)